tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java

```diff
@@ -36,11 +36,13 @@
   /** The maximum value for the size value for the test object. */
   private static final int MAX_SIZE = 30;
   /** The duration for each test. */
-  private static final int TEST_CASE_DURATION_MS = 5000;
+  private static final int TEST_CASE_DURATION_MS = 30000;
```

Comment:
This should probably still be 5 seconds for now, so integration tests don't get too long.